packages/cocos/package.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "@ton/cocos-sdk",
+    "version": "0.0.1-beta.1",
+    "files": ["./dist"],
+    "main": "dist/cocos/src/index.js",
+    "types": "dist/cocos/src/index.d.ts",
+    "module": "dist/cocos/src/index.js",
+    "scripts": {
+        "dev": "tsc --watch",
+        "build": "tsc",
+        "build:webpack": "webpack --mode production && tsc --declaration --emitDeclarationOnly"
```

Comment:
Should we still run `tsc --declaration --emitDeclarationOnly` if we use a plugin to generate declaration?

Reply:
I found that the declaration file generated by TypescriptDeclarationPlugin of webpack has some problems. In fact, this plugin is not needed. Maybe we should remove TypescriptDeclarationPlugin in webpack. There is no need to execute tsc --declaration --emitDeclarationOnly. webpack will generate the correct